docs/documentation/getting-started/configuration/index.markdown

```diff
@@ -68,6 +68,7 @@ The following variables are settable:
   * Example: `set :repo_url, 'git@example.com:me/my_repo.git'` for a git repo located in /home/git/me
   * Hint #1: to access a repo on a machine using a non-standard ssh port: `set :repo_url, 'ssh://git@example.com:30000/~/me/my_repo.git'`
   * Hint #2: when using :svn and branches, declare the repo_url like this: `set :repo_url, -> { "svn://myhost/myrepo/#{fetch(:branch)}" }`
+  * Warning: if you move your repository to a new URL and change this variable, already deployed remote servers won't reflect this change automatically, you have to manually re-configure the repository on the remote servers (in path determined by `:repo_path`) or delete it and let Capistrano recreate it on the next deploy using the updated URL.
```

Comment:
That's the idea I had when I went into it. Then I thought about it and realized that you could only do with changing the repo (`git origin set-url`) which might be worth considering if your repo is huge and you want to avoid re-downloading all of it. On top of that I noticed that `:repo_path` is a configurable variable so saying `rm -rf repo` might not apply.

As a compromise, I'd go with `... or delete it (rm -rf repo in the default setup) and let Capistrano...`